channel.go

```diff
@@ -132,7 +133,38 @@ func (c *vSockChannel) setup() error {
 }
 
 func (c *vSockChannel) wait() error {
-	return nil
+	if vsockHostPort == uint32(0) {
+		return fmt.Errorf("fail to get vsock host port %d", vsockHostPort)
```

Comment:
Please mention agent.vhost_vsock_port in this error message so it's clear that this parameter is missing.